### PR TITLE
[DO NOT MERGE] build/configs/rtl8720e: Optimize the PSRAM size

### PR DESCRIPTION
**Pull request by ZhenBei-Sin**
For RTL8720E: 
Changed .ld file
Updated km4_boot_all.bin & km4_boot_all_tz.bin & km4_image3_all.bin & cmse_implib.lib

### CODE REVIEW

File: build/configs/rtl8720e/scripts/rlx8720e_img2_is.ld [hunk 1]
```diff
@@ -67,9 +67,9 @@ MEMORY
 	KM4_MSP_RAM_S (rw) :                    ORIGIN = 0x30008000, LENGTH = 0x3000A000 - 0x30008000	/* KM4 MSP_S RAM: 8k */
 	KM4_BOOTLOADER_RAM_S (rwx) :            ORIGIN = 0x3000A020, LENGTH = 0x30012000 - 0x3000A020	/* KM4 BOOT Loader RAM: 32k */
 	KM4_FLOADER_RAM_S (rwx) :               ORIGIN = 0x3000A020, LENGTH = 0x30020000 - 0x3000A020	/* KM4 ImgTool Flash Loader RAM: 88k */
-	KM4_BD_RAM_NSC (rwx) :                  ORIGIN = 0x60347020, LENGTH = 0x60348000 - 0x60347020	/* KM4 BD RAM NSC: 4k */
-	KM4_BD_RAM_ENTRY (rwx) :                ORIGIN = 0x60348000, LENGTH = 0x6034C000 - 0x60348000	/* KM4 BD RAM ENTRY: 16k */
-	KM4_BD_RAM_S (rwx) :                    ORIGIN = 0x7034C020, LENGTH = 0x7039C000 - 0x7034C020	/* KM4 BD RAM S: 320k */
+	KM4_BD_RAM_NSC (rwx) :                  ORIGIN = 0x60351020, LENGTH = 0x60352000 - 0x60351020	/* KM4 BD RAM NSC: 4k */
```
Comment (sangam-swami) [inline build/configs/rtl8720e/scripts/rlx8720e_img2_is.ld:70]: can you please change commit message to build/configs/rtl8720e for clearly knowing which board this PR for

**Comment by thapav** [comment]
Have you verified build and booting with all three configurations ?
hello,tc & laodable_apps ?

**Comment by ZhenBei-Sin** [comment]
> Have you verified build and booting with all three configurations ? hello,tc & laodable_apps ?

Verification (compare btw this #PR5622 and RTL8720E_Support br c6d2d93b):
hello -> OK build & bootup - Additional tested the security test code as well
loadable_apps -> OK build but faced the same issue will be hanging there
tc -> Failed to build it was same with br c6d2d93b

I had attached the logfile and the JPG to refer.

tc_gitSHA
[hello.log](https://github.com/Samsung/TizenRT/files/9450877/hello.log)
[lodable_apps.log](https://github.com/Samsung/TizenRT/files/9450870/lodable_apps.log)
tc_build_failed

**Comment by sunghan-chang** [comment]
@ZhenBei-Sin Could you check belows?
1. Did you run clean and build when you change the build configuration like hello -> tc?
2. Did you flash binaries using ./dbuild.sh? Did you do for bootparam.bin as well?

**Comment by sunghan-chang** [comment]
@ZhenBei-Sin If you did above, could you erase entire flash before downloading for loadable_apps? The log shows bootparam is wrong.

**Comment by ZhenBei-Sin** [comment]
> @ZhenBei-Sin If you did above, could you erase entire flash before downloading for loadable_apps? The log shows bootparam is wrong.

Hi @sunghan-chang , after ERASE ALL -> use  ./dbuild.sh DOWNLOAD.
The loadable_apps is work fine in this PR branch, you may refer to the attached logfile.

[loadable_apps_OK.log](https://github.com/Samsung/TizenRT/files/9475193/loadable_apps_OK.log)

Conclusion: 
hello -> OK build & bootup - Additional tested the security test code as well
loadable_apps -> OK build & bootup
tc -> Failed to build it was same error with br c6d2d93

Any idea for tc? It is no related the changes from this PR as the issue faced on the previous br also.

**Comment by sunghan-chang** [comment]
@ZhenBei-Sin It's the issue with not enough size of KM4_BD_RAM for tc build configuration. I think you need to move some data into another region. Anyway it was ok before. I think because many changes were merged, it needs more data, bss and so on. In other words, bsp is getting bigger than when we checked.

**Comment by ZhenBei-Sin** [comment]
Hi @sunghan-chang,

Please **do not merge** this PR as conflict with the PR5632 : os/board/rtl8720e/src: Correct MPU setting and disable ROM APIs call in TrustZone.

Thank You.

**Pull request closed without merging**